.github/workflows/ci.yml

```diff
@@ -11,7 +11,7 @@ jobs:
       - uses: actions/checkout@v2
```

Comment:
idk if we want to do it in this PR, but we'll want type tests for the public types added to here -- for some matrix we want to support.

I've been trying to target TS 4.5+ where I can

Reply:
Added